### PR TITLE
[ts] tweak patching docs based on Flick.tech feedback

### PR DESCRIPTION
**Pull request by sw-yx**
## What does this PR do?

as above

### CODE REVIEW

File: docs/typescript/patching.md [hunk 1]
```diff
@@ -14,7 +14,55 @@ Before you explore our patching/versioning API, check if your needs can be addre
...
+export async function myWorkflow({ force }): Promise<void> {
+  await sleep('1 days');
+  console.log('force', force);
+}
+
+// to v2
+export async function myWorkflow(payload): Promise<void> {
+  let force = payload.force;
+
+  setListener(updatePayloadSignal, (newPayload) => {
```
Comment (bergundy) [inline docs/typescript/patching.md:39]:
```suggestion
  setHandler(updatePayloadSignal, (newPayload) => {
```

File: docs/typescript/patching.md [hunk 2]
```diff
@@ -14,7 +14,55 @@ Before you explore our patching/versioning API, check if your needs can be addre
...
+    force = newPayload.force;
+  });
+
+  await sleep('1 days');
+  console.log('force', force);
+}
+```
+
+This is an example of code that will need patching:
```
Comment (bergundy) [inline docs/typescript/patching.md:48]: This is more an in your face example of how npm packages might break determinism.

File: docs/typescript/patching.md [hunk 3]
```diff
@@ -101,3 +149,5 @@ If while we're deploying `v2deprecatedpatch` (below) there are still live Worker
 
 Upgrading Workflow dependencies (such as ones installed into `node_modules`) _might_ break determinism in unpredictable ways.
 It is highly recommended to use a lock file (`package-lock.json` or `yarn.lock`) in order to fix Workflow dependency versions and gain control of when they're updated.
+
+Be aware that any of your dependencies might do something non-deterministic, such as filesystem access or network calls.
```
Comment (bergundy) [inline docs/typescript/patching.md:153]: This is not true because WF runs in a sandbox

File: docs/typescript/patching.md (hunk 2 above)
Comment (sw-yx) [reply docs/typescript/patching.md:48]:
```suggestion
This is an example of how npm packages can break determinism:
```

File: docs/typescript/patching.md (hunk 3 above)
Comment (sw-yx) [reply docs/typescript/patching.md:153]:
```suggestion
```

**Pull request merged**